config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml

```diff
@@ -6,6 +6,9 @@ periodics:
     repo: kubernetes
     base_ref: windows-up
     path_alias: k8s.io/kubernetes
+  - org: kubernetes
+    repo: release
+    base_ref: master
```

Comment:
`path_alias: k8s.io/release` is also needed I think

Reply:
Done, but I just missed the run of the hour :(